config/i18n.js

```diff
@@ -1,5 +1,13 @@
-export default {
+const clientI18n = {
   langPickerRender: true,
   langList: ['en-US', 'ru-RU'],
   fallbackLng: 'en-US'
 };
+
+const serverI18n = {
+  locales: ['en', 'ru'],
+  directory: process.cwd() + '/src/modules/i18n/locales',
+  cookie: 'locale'
```

Comment:
Its okay to have parameter for cookie name, but you should use this parameter both for client-side i18n and for server-side i18n.